pkg/client/client.go

```diff
@@ -968,6 +970,12 @@ func (c *Client) ValidatePrometheus(ctx context.Context, promNsName types.Namesp
 	}, WithPollInterval(10*time.Second))
 
 	if pollErr != nil {
+		s, _ := c.kclient.AppsV1().StatefulSets(promNsName.Namespace).Get(ctx, "prometheus-k8s", metav1.GetOptions{})
+		generationAttemps := s.Status.ObservedGeneration - lastGeneration
```

Comment:
```suggestion
		generationAttempts := s.Status.ObservedGeneration - lastGeneration
```
(nit)

Reply:
+1 on the nit, actually we should have one linter for it :D

Reply:
I believe the `misspell` and the `misspell-fix` targets work for us here, do you want to fail the CI if it errors out?